app/api/domains/cho.py

```diff
@@ -84,20 +84,100 @@
 @router.get("/")
 async def bancho_http_handler():
     """Handle a request from a web browser."""
+    new_line = "\n"
+    matches = list(
+        filter(lambda match: isinstance(match, Match), app.state.sessions.matches),
+    )
```

Comment:
are they ever not of instance `Match`?

Reply:
[there's None](https://github.com/arily/bancho.py/blob/9078c4f68e22d6b8f096baeef9d918d48803e2e4/app/objects/collections.py#L175)

Reply:
oh, probably better to check for that

Reply:
```py
matches = [m for m in app.state.sessions.matches if m is not None]
```

Reply:
f-strings below will throw error if None is not filtered out, tested locally

Reply:
yea you 100% need to remove the nones, the `if m is not None` does the filtering in my example

Reply:
updated